source/manual/nagios-nrpe-connection-failures.html.md

```diff
@@ -22,10 +22,13 @@ tries the check again (within a minute or two). If the errors persist for
 longer than a few minutes there may be a genuine issue and you should
 investigate.
 
+Note: This example uses hostnames for Carrenza, and will not work on AWS.
+For AWS you'll need to use the hosts from `govuk_node_list`.
```

Comment:
Do you think it would be beneficial to provide more context about this, or maybe a link to the relevant documentation?

Reply:
What did you have in mind?

Reply:
Maybe a link to [this](https://docs.publishing.service.gov.uk/manual/howto-ssh-to-machines-in-aws.html)?

Reply:
@AlanGabbianelli Updated with the link.